.env.example

```diff
@@ -0,0 +1,8 @@
+GO_FOX_HOST=localhost:3000
```

Comment:
Why

Reply:
Thanks @spencerldixon I put this in because it was present on a different project which I was learning from but I can remove it if we don't need it. Actually I noticed now that Future of Conservation does not have this either.